rate: 100.0

model:

  translation:

    n_states: 12
    n_inputs: 3

    A: [1, 0.01, 0.00005,       0, 0,   0,        0,       0, 0,    0,       0,       0,
        0,    1,    0.01, 0.00005, 0,   0,        0,       0, 0,    0,       0,       0,
        0,    0,       1,    0.01, 0,   0,        0,       0, 0,    0,       0,       0,
        0,    0,       0,       1, 0,   0,        0,       0, 0,    0,       0,       0,
        0,    0,       0,       0, 1, 0.01, 0.00005,       0, 0,    0,       0,       0,
        0,    0,       0,       0, 0,    1,    0.01, 0.00005, 0,    0,       0,       0,
        0,    0,       0,       0, 0,    0,       1,    0.01, 0,    0,       0,       0,
        0,    0,       0,       0, 0,    0,       0,       1, 0,    0,       0,       0,
        0,    0,       0,       0, 0,    0,       0,       0, 1, 0.01, 0.00005,       0,
        0,    0,       0,       0, 0,    0,       0,       0, 0,    1,    0.01, 0.00005,
        0,    0,       0,       0, 0,    0,       0,       0, 0,    0,       1,    0.01,
        0,    0,       0,       0, 0,    0,       0,       0, 0,    0,       0,       1]

    B: [0, 0, 0,
        0, 0, 0,
        0, 0, 0,
        0.01, 0, 0,
        0, 0, 0,
        0, 0, 0,
        0, 0, 0,
        0, 0.01, 0,
        0, 0, 0,
        0, 0, 0,
        0, 0, 0,
        0, 0, 0.01]

  heading:

    n_states: 4
    n_inputs: 1

    A: [1, 0.01, 0.00005,       0,
        0,    1,    0.01, 0.00005,
        0,    0,       1,    0.01,
        0,    0,       0,       1]

    B: [0,
        0,
        0,
        0.01]